src/common/commonutils/SshUtils.c

```diff
@@ -612,9 +619,9 @@ static int SetSshWarningBanner(unsigned int desiredBannerFileAccess, const char*
     return status;
 }
 
-static int IncludeSshRemediationConfFile(void* log)
+static int IncludeRemediationSshConfFile(void* log)
 {
-    const char* confFolder = "/etc/ssh/sshd_config.d";
+    const char* etcSshSshdConfigD = "/etc/ssh/sshd_config.d";
```

Comment:
nit: not crazy about the name, can it just have one instance of ssh? `etcSshdConfigD`

Reply:
This is used in many places here. The file name is converted to camel casing entirely. It's an internal name only and for me it's clearer this way, personal views may differ.